textile/features.textile

```diff
@@ -1655,6 +1656,17 @@ h4. DevicePushDetails
 * @(PCP3)@ recipient - a map of string key/value pairs containing details of the push transport and address
 * @(PCP4)@ state - the state of the push registration, one of  @Active@, @Failing@, @Failed@
 
+h3(#introspection). Client library introspection
+
+h4. ClientLibraryInfo
```

Comment:
I am not a fan of abbreviations in API names (I know we already have some like `Params`, but I would prefer not to add more going forwards), so would prefer expansion to `ClientLibraryInformation` - but that is starting to get a little unwieldy. Perhaps `Library` does not need to be stated, given context of use? And, also, even the word `Information` feels like it's grappling at straws in terms of adding value - these are static methods/properties, so is it not obvious that this is a 'utility' class? 🤔 

So, I wonder whether a shorter name, perhaps a little more aligned with existing `RestClient` and `RealtimeClient` names might be `ClientRuntime`?

Reply:
Naming is horrible, isn't it?! I have made your suggested change, and am happy to hear others’ opinions too.

Reply:
I definitely agree that we don't need `Library` there. I'm not sure how I feel about "Runtime" though, I think someone looking through an SDK API will probably be more confused by that than if we just call it `ClientInformation`, @QuintinWillison wdyt?

Reply:
Fair point around specificity. I would also be happy with `ClientInformation`.

Reply:
Changed it to `ClientInformation`.

Reply:
How about `AgentInformation`, since `ClientInformation` sounds _a bit_ fishy @QuintinWillison @owenpearson

Reply:
I'm going to leave this one for the three of you to battle out haha. @QuintinWillison @owenpearson did you have any opinions on @maratal’s question? I'm keen to get this PR merged pretty soon.

Reply:
I understand where @maratal is coming from, however our context is an Ably SDK and this means that 'client' should be well understood as having come from 'client library'. I also think that 'Agent' start to be too specific again, perhaps inferring that the APIs here-presented relate only to the agents headers sent to the Ably service on connection. I think we should stick with `ClientInformation` and get this PR landed!